chainer/function_hooks/timer.py

```diff
@@ -61,3 +83,44 @@ def backward_postprocess(self, function, in_data, out_grad):
     def total_time(self):
         """Returns total elapsed time in seconds."""
         return sum(t for (_, t) in self.call_history)
+
+    def summary(self):
+        """A summary of time profiling in functions.
+
+        Returns:
+            Returns a summarized dictionary whose keys are function names and
+            values are dictionaries of `elapsed_time` and `occurrrence`.
```

Comment:
I'm sorry I annotated a wrong line.
What should be verb form is the summary line (*A summary of ...*).
The body of "Returns" section is okay to be noun form.